### ISSUE TITLE
Incorrect parser error when calling Function constructor

### ISSUE THREAD

**Issue by galpeter** [Samsung/jerryscript#364]
The testcase:
```js
var func = new Function("return 2");
```

reports the following error:
```
        ^
ERROR: Ln 1, Col 9: Expected either ';' or newline token
```

However this should not cause any errors.

**Comment by ruben-ayrapetyan** [comment]
On current master the test cases finishes successfully.

**Comment by ruben-ayrapetyan** [comment]
Fixed by #373
